.github/workflows/pr-require-label.yml

```diff
@@ -1,17 +0,0 @@
-name: Enforce PR labels
-
-on:
-  pull_request:
-    types: [labeled, unlabeled, opened, edited, synchronize]
-  workflow_run:
-    workflows: ["Pull Request Labeler"]
-    types:
-      - completed
```

Comment:
This very condition was supposed to re-run the "Enforce PR labels" after "Pull Request Labeler". At least, that's how it worked in my fork.

So what's different? In the PR discussion a [loop prevention](https://github.com/immich-app/immich/pull/10081#issuecomment-2158738753) is mentioned. Does that prevent the desired behaviour?

Reply:
Not sure really, I think we basically decided it wasn't worth investigating when the labels aren't actually "mandatory" they're just a really nice help and the other action should catch most cases anyway

Reply:
Fair enough. When I implemented it this way, I felt a little dirty about the dependency across workflows. Less complexity is a good thing, too.